src/openrct2/entity/Guest.cpp

```diff
@@ -440,7 +440,7 @@ static void PeepRideIsTooIntense(Guest* peep, Ride& ride, bool peepAtRide);
 static void PeepResetRideHeading(Guest* peep);
 static void PeepTriedToEnterFullQueue(Guest* peep, Ride& ride);
 static int16_t PeepCalculateRideSatisfaction(Guest* peep, const Ride& ride);
-static void PeepUpdateFavouriteRide(Guest* peep, const Ride& ride);
+static void PeepUpdateFavouriteRide(Guest& peep, const Ride& ride, const uint8_t satisfaction);
```

Comment:
It’s optional, but since we’re modifying the function anyway, I think it’s a good idea to rename it to `GuestUpdateFavouriteRide`.